.github/workflows/go-linter.yml

```diff
@@ -0,0 +1,41 @@
+name: Go linter
+
+on:
+  workflow_call:
+    inputs:
+      ref:
+        description: 'Ref to test [default: latest master; examples: 0a4ff9d3e4a9ab432fd5812eb18c98e03b5a7432'
+        required: false
+        default: ''
+      workdir:
+        description: 'Working directory'
+        required: false
+        default: '.'
+
+jobs:
+  linter:
+    name: Lint
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.ref }}
+
+      - name: Checkout .github repository
+        uses: actions/checkout@v4
+        with:
+          repository: nspcc-dev/.github
+          path: nspcc-gh
+
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: ${{ github.event.inputs.workdir }}/go.mod
```

Comment:
For neo-bench can I setup custom `workdir` via job argument? It has an unusual project structure.

Reply:
That's exactly why we have it.